src/environment.rs

```diff
@@ -130,7 +130,7 @@ pub async fn get_up_to_date_prefix(
     sanity_check_project(project)?;
 
     // Start loading the installed packages in the background
-    let prefix = Prefix::new(project.root().join(".pixi/env"))?;
+    let prefix = Prefix::new(project.environment_dir())?;
```

Comment:
Please make sure all `environment_dir()`'s are renamed to `pixi_dir(). Like in this file on line 83.

Reply:
I'm confused.

I already checked, and I cannot find any more matches for `".pixi/env"` in the code.

I also checked for `pixi_dir()` matches and could not find any.

Do you mean I should renamed `environment_dir` to `pixi_dir`? But my understanding is that `prefix` should point to an env (`.pixi/env`) rather than a pixi dir (`.pixi/`).

Let me know if I got something wrong.

Reply:
The `.pixi` folder is the one we use for the `prefix_file`.  Around this logic is also the only location we use the `environment_dir()`. Sorry for the confusion. The other location would be `src/cli/add.rs:113` where this needs to be changed.

Reply:
Ok, got it now. I think I missed it when you said `Like in this file on line 83.` which got me confused.

 I think we should be good now.